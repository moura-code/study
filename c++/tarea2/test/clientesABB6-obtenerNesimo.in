crearClientesABB
crearCliente 15 Rufino Garcia 21
agregarAClientesABB
obtenerNesimoClienteClientesABB 1
crearCliente 3 Zafiro Fernandez 25
agregarAClientesABB
obtenerNesimoClienteClientesABB 1
obtenerNesimoClienteClientesABB 2
crearCliente 25 Casimira Sanchez 26
agregarAClientesABB
crearCliente 22 Zafiro Sanchez 33
agregarAClientesABB
crearCliente 20 Rufino Gonzalez 23
agregarAClientesABB
crearCliente 19 Esmeralda Ruiz 32
agregarAClientesABB
crearCliente 21 Esmeralda Garcia 58
agregarAClientesABB
crearCliente 2 Rufino Lopez 28
agregarAClientesABB
crearCliente 1 Valquiria Ruiz 27
agregarAClientesABB
crearCliente 6 Zafiro Rodriguez 42
agregarAClientesABB
obtenerNesimoClienteClientesABB 10
obtenerNesimoClienteClientesABB 9
obtenerNesimoClienteClientesABB 8
obtenerNesimoClienteClientesABB 7
obtenerNesimoClienteClientesABB 6
obtenerNesimoClienteClientesABB 1
obtenerNesimoClienteClientesABB 2
obtenerNesimoClienteClientesABB 3
obtenerNesimoClienteClientesABB 4
obtenerNesimoClienteClientesABB 5
liberarClientesABB
Fin
