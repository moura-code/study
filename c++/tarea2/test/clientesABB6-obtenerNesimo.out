Bienvenido al programa principal de Mercado FINGer. Por favor ingrese su comando:
1> El clientesABB ha sido creado de forma exitosa.
2> Cliente creado de forma exitosa.
3> Se ha agregado el cliente al clientesABB de forma exitosa.
4> Cliente nro 1 del clientesABB:
Cliente Rufino Garcia
Id: 15
Edad: 21
5> Cliente creado de forma exitosa.
6> Se ha agregado el cliente al clientesABB de forma exitosa.
7> Cliente nro 1 del clientesABB:
Cliente Zafiro Fernandez
Id: 3
Edad: 25
8> Cliente nro 2 del clientesABB:
Cliente Rufino Garcia
Id: 15
Edad: 21
9> Cliente creado de forma exitosa.
10> Se ha agregado el cliente al clientesABB de forma exitosa.
11> Cliente creado de forma exitosa.
12> Se ha agregado el cliente al clientesABB de forma exitosa.
13> Cliente creado de forma exitosa.
14> Se ha agregado el cliente al clientesABB de forma exitosa.
15> Cliente creado de forma exitosa.
16> Se ha agregado el cliente al clientesABB de forma exitosa.
17> Cliente creado de forma exitosa.
18> Se ha agregado el cliente al clientesABB de forma exitosa.
19> Cliente creado de forma exitosa.
20> Se ha agregado el cliente al clientesABB de forma exitosa.
21> Cliente creado de forma exitosa.
22> Se ha agregado el cliente al clientesABB de forma exitosa.
23> Cliente creado de forma exitosa.
24> Se ha agregado el cliente al clientesABB de forma exitosa.
25> Cliente nro 10 del clientesABB:
Cliente Casimira Sanchez
Id: 25
Edad: 26
26> Cliente nro 9 del clientesABB:
Cliente Zafiro Sanchez
Id: 22
Edad: 33
27> Cliente nro 8 del clientesABB:
Cliente Esmeralda Garcia
Id: 21
Edad: 58
28> Cliente nro 7 del clientesABB:
Cliente Rufino Gonzalez
Id: 20
Edad: 23
29> Cliente nro 6 del clientesABB:
Cliente Esmeralda Ruiz
Id: 19
Edad: 32
30> Cliente nro 1 del clientesABB:
Cliente Valquiria Ruiz
Id: 1
Edad: 27
31> Cliente nro 2 del clientesABB:
Cliente Rufino Lopez
Id: 2
Edad: 28
32> Cliente nro 3 del clientesABB:
Cliente Zafiro Fernandez
Id: 3
Edad: 25
33> Cliente nro 4 del clientesABB:
Cliente Zafiro Rodriguez
Id: 6
Edad: 42
34> Cliente nro 5 del clientesABB:
Cliente Rufino Garcia
Id: 15
Edad: 21
35> ClientesABB liberado con exito.
36> Fin.
